scripts/download-rc.js

```diff
@@ -67,9 +67,9 @@ const downloadRC = () => {
         const subprocess = spawnSync('mvn',
             [
                 '-q',
-                'dependency:get',
+                'org.apache.maven.plugins:maven-dependency-plugin:2.10:get',
```

Comment:
Isn't the latest 3.6.0?

Reply:
> Isn't the latest 3.6.0?

@srknzl yes, but this is the latest version that supports the `-Ddest` argument.